backend/graminsta/core/views.py

```diff
@@ -47,28 +47,17 @@ def get(self, request):
         Returns
         --------------
         response: json format
-            username of all users that satisfy the requirements
+            displayed name of all users that satisfy the requirements
         """
-        username = get_all_username()
+        all_users = get_all_users()
+        all_users_name = []
+        separator = ","
+        for user in all_users:
+            all_users_name.append(user.first_name + " " + user.last_name)
```

Comment:
Why not just use `UserInfoSerializer` to serialize these `User` objects. If has all the information you need

Reply:
The front-end only need displayed username, so i think maybe we should just return it instead of all user objects?